src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -351,6 +351,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
 
   <ItemGroup>
+    <RuntimeHostConfigurationOption Include=" System.Runtime.InteropServices.Marshal.IsBuiltInComSupported"
```

Comment:
These are all alphabetized by the Include. Can you sort the new `System.Runtime.....` item into the appropriate place?

Also, this shouldn't have a blank space at the beginning of the name.